Eastern-Research-Group/mywaterway
# Feature/237 waterbody report images

## Related Issues:
* [HMW-237](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-237)

## Main Changes:
* Added the capability to display an image in the popup of the Waterbody Report page: if a link to a photo is provided in the corresponding assessment unit data, and it is one of the allowed image types, it will show in the popup for the waterbody.
* Currently, the first photo found is used, though the ATTAINS data may contain several.

## Steps To Test:
1. Only one waterbody with assessment data has a photo uploaded to ATTAINS: it can be found at [http://localhost:3000/waterbody-report/DOEE/DCANA00E_01/2022](http://localhost:3000/waterbody-report/DOEE/DCANA00E_01/2022)
2. Click the waterbody graphic on the map to confirm that the photo is visible in the popup
3. Check the popup on any other Waterbody Report page to confirm that the popup still loads properly without an image

## TODO:
- [ ] The photo for the waterbody linked above is hosted on the _demo_ ATTAINS endpoint, so the `proxyFetch` function and the `serviceUrlDev` service are being used. Once photos are available on staging or production, these can be changed to `fetchCheck` and `serviceUrl`, respectively.
